package-lock.json

```diff
@@ -20110,7 +20110,7 @@
 		"app-root-dir": {
 			"version": "1.0.2",
 			"resolved": "https://registry.npmjs.org/app-root-dir/-/app-root-dir-1.0.2.tgz",
-			"integrity": "sha512-jlpIfsOoNoafl92Sz//64uQHGSyMrD2vYG5d8o2a4qGvyNCvXur7bzIsWtAC/6flI2RYAp3kv8rsfBtaLm7w0g==",
+			"integrity": "sha1-OBh+wt6nV3//Az/8sSFyaS/24Rg=",
```

Comment:
Are these hash changes an artefact from a previous commit when we added `change-case`?

Doesn't look like there are any new package.json libs in this PR anymore. Can it the lock file be reverted?

Reply:
Ah yep this can be reverted.